src/qt/forms/aboutdialog.ui

```diff
@@ -81,8 +59,11 @@
        </item>
        <item>
         <widget class="QLabel" name="versionLabel">
+         <property name="cursor">
+          <cursorShape>IBeamCursor</cursorShape>
+         </property>
          <property name="text">
-          <string notr="true">v0.1.0</string>
+          <string notr="true">0.3.666-beta</string>
```

Comment:
It's actually just used as preview text in Qt Designer.

I can change it back if needed but this seemed more appropriate. I just realised though I could have improved it yet again with something like 0.3.6.9-beta but to be honest it doesn't actually effect anything in production so I don't mind.